[IMP] CRM: multiple sales teams 17 (#8961)
odoo/documentation: content/applications/sales/crm/pipeline/multi_sales_team.rst

Multiple sales teams
====================

The *Multi Teams* feature in Odoo's *CRM app* allows salespeople to be assigned to multiple sales
teams. Multiple teams can be created and manage in one dashboard, each with their own pipeline for
leads and opportunities.

Enable multi teams
==================

To allow salespeople to be assigned to more than one sales team, the :guilabel:`Multi Teams` setting
needs to be enabled. First, navigate to :menuselection:`CRM --> Configuration --> Settings`. Under
the :guilabel:`CRM` section, tick the checkbox labeled :guilabel:`Multi Teams`. Then, click
:guilabel:`Save` at the top-left of the page.

.. image:: multi_sales_team/multi-team-setting.png
   :align: center
   :alt: The Multi Teams setting in the CRM app.

.. _crm/assignment-rules:

Enable assignment rules
=======================

To periodically :doc:`assign leads <../track_leads/lead_scoring>` based on rules, the

> :guilabel:`Rule-Based Assignment` feature needs to be enabled. Navigate to :menuselection:`CRM -->

Reviewer: ```suggestion
:guilabel:`Rule-Based Assignment` feature needs to be enabled. Navigate to :menuselection:`CRM app -->
```

Configuration --> Settings`. Under the :guilabel:`CRM` section, tick the checkbox labeled
:guilabel:`Rule-Based Assignment`.

> This causes a new field, :guilabel:`Running` to appear. Select an option from the drop-down to

Reviewer: ```suggestion
This causes a new field, :guilabel:`Running`, to appear. Select an option from the drop-down menu in this field to
```

manage how assignments are triggered.

> The rule-based assignment feature can be set to run :guilabel:`Manually`, meaning an Odoo user must

Reviewer: ```suggestion
The :guilabel:`Rule-Based Assignment` feature can be set to run :guilabel:`Manually`, meaning an Odoo user must
```

manually trigger the assignment, or :guilabel:`Repeatedly`, meaning Odoo automatically triggers the
assignments.

Selecting :guilabel:`Repeatedly` causes additional fields to appear. Use the :guilabel:`Repeat
every` fields to select how frequently the rules should be triggered. The :guilabel:`Next Run` field

> automatically updates, thought it can be manually updated.

Reviewer: ```suggestion
automatically updates, though it can be manually updated, if needed.

Once all desired configurations are complete, click :guilabel:`Save`.
```

.. image:: multi_sales_team/rule-based-assignment.png
   :align: center
   :alt: The Rule-Based Assignment setting enabled in the CRM app.

.. seealso::

>    For more information on assigning leads, see :doc:`assign leads <../track_leads/lead_scoring>`.

Reviewer: ```suggestion
   For more information on assigning leads, see the :doc:`assign leads <../track_leads/lead_scoring>` documentation.
```

> Create a new sales team

Reviewer: ```suggestion
Create sales teams
```

=======================

> To create a new sales team, go to :menuselection:`CRM --> Configuration --> Sales Teams`, then

Reviewer: ```suggestion
To create a new sales team, go to :menuselection:`CRM app --> Configuration --> Sales Teams`, then
```

click :guilabel:`New`.

> On the blank record enter a name in the :guilabel:`Sales Team` field.

Reviewer: ```suggestion
On the blank sales team form, enter a name in the :guilabel:`Sales Team` field.
```

Next, select a :guilabel:`Team Leader` from the drop-down list.

> Set an :guilabel:`Email Alias` to automatically generate a lead/opportunity for this sales team when

Reviewer: ```suggestion
Set an :guilabel:`Email Alias` to automatically generate a lead/opportunity for this sales team whenever
```

a message is sent to that unique email address. Choose whether to accept emails from

> :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or :guilabel:`Followers Only`.

Reviewer: I'm also seeing an option for 'Authenticated Employees'...might be worth a quick double-check on that during your revision

Multiple sales teams
====================

The *Multi Teams* feature in Odoo's *CRM* app allows salespeople to be assigned to multiple sales
teams. Multiple teams can be created and managed in one dashboard, each with their own pipeline for
leads and opportunities.

Enable multi teams
==================

To allow salespeople to be assigned to more than one sales team, the :guilabel:`Multi Teams` setting
needs to be enabled. First, navigate to :menuselection:`CRM app --> Configuration --> Settings`.
Under the :guilabel:`CRM` section, tick the checkbox labeled :guilabel:`Multi Teams`. Then, click
:guilabel:`Save` at the top-left of the page.

.. image:: multi_sales_team/multi-team-setting.png
   :align: center
   :alt: The Multi Teams setting in the CRM app.

.. _crm/assignment-rules:

> Enable assignment rules

Reviewer 2: It's not clear to me how this section is exactly tied to the Multi Teams feature, and we already have a doc for the assignment rules. Can we more clearly make that connection? Otherwise if it's relevant in a tertiary way we can just do a **seealso**, no?

=======================

To periodically :doc:`assign leads <../track_leads/lead_scoring>` based on rules, the
:guilabel:`Rule-Based Assignment` feature needs to be enabled. Navigate to :menuselection:`CRM app
--> Configuration --> Settings`. Under the :guilabel:`CRM` section, tick the checkbox labeled
:guilabel:`Rule-Based Assignment`.

This causes a new field, :guilabel:`Running`, to appear. Select an option from the drop-down menu in
this field to manage how assignments are triggered.

> The :guilabel:`Rule-Based Assignment` feature can be set to run :guilabel:`Manually`, meaning an

Reviewer 2: Suggestion for specificity: 
It's implied right here to run **Manually** on the **Settings > CRM** page, however I would clearly specify where it's possible to run the rules manually, and how to do it, for users who wish to do this.

Odoo user must manually trigger the assignment, or :guilabel:`Repeatedly`, meaning Odoo
automatically triggers the assignments.

Selecting :guilabel:`Repeatedly` causes additional fields to appear. Use the :guilabel:`Repeat
every` fields to select how frequently the rules should be triggered. The :guilabel:`Next Run` field
automatically updates, though it can be manually updated, if needed.

Once all desired configurations are complete, click :guilabel:`Save`.

.. image:: multi_sales_team/rule-based-assignment.png
   :align: center
   :alt: The Rule-Based Assignment setting enabled in the CRM app.

.. seealso::
   For more information on assigning leads, see the :doc:`assign leads
   <../track_leads/lead_scoring>` documentation.

> Create sales teams

Reviewer 2: Similar feedback here...I would think the basics of creating and organizing sales teams in the CRM app would ideally be it's own doc, namely these two next sections starting at line 52.

As it is, this doc is coming across as a bit of a hybrid covering multiple topics and not necessarily how to manage multiple sales teams.

---------------------------

> To add team members, click :guilabel:`Add` under the :guilabel:`Members` tab when editing the sales

Reviewer: ```suggestion
To add team members, click :guilabel:`Add`, under the :guilabel:`Members` tab, when editing the sales
```

team's configuration page. This opens a :guilabel:`Create Sales Team Members` pop-up window.

> Select a user from the :guilabel:`Salesperson` drop-down list to add to the team. To prevent this

Reviewer: ```suggestion
Select a user from the :guilabel:`Salesperson` drop-down list to add them to the team. To prevent this
```

The :guilabel:`Leads (30 days)` field tracks how many leads the salesperson has been assigned in the
past thirty days, and the maximum number of leads they should be assigned. To edit the maximum
number of leads this salesperson can be assigned, enter the new number in the :guilabel:`Leads (30
days)` field.

.. tip::
   :doc:`Assignment rules <../track_leads/lead_scoring>` can be configured for individual
   salespeople using the :guilabel:`Domain` section.

.. image:: multi_sales_team/create-team-members.png
   :align: center
   :alt: The Create Sales Team Members pop-up window.

Click :guilabel:`Save & Close` when finished, or :guilabel:`Save & New` to add additional members.

.. note::
   If the :guilabel:`Rule-Based Assignment` feature has **not** been enabled, clicking
   :guilabel:`Add` under the :guilabel:`Members` tab opens a :guilabel:`Add:Salespersons` pop-up
   window. Tick the checkbox to the far-left of the salesperson to be added to the team, then click
   :guilabel:`Select`.

   .. image:: multi_sales_team/add-salespersons.png
      :align: center
      :alt: The Add:Salespersons pop-up window in the CRM app.

Sales team dashboard
====================

> To view the sales team dashboard, go to :menuselection:`CRM --> Sales --> Teams`. Any team a user

Reviewer: ```suggestion
To view the sales team dashboard, go to :menuselection:`CRM app --> Sales --> Teams`. Any team a user
```
